index.js

```diff
@@ -0,0 +1,31 @@
+let theBooks = [];
+
+const addButton = document.querySelector('.button');
+addButton.addEventListener('click', () => {
+  const bookContainer = document.createElement('div');
+  const titleBook = document.createElement('h5');
+  const authorBook = document.createElement('h5');
+  const removeButton = document.createElement('button');
+  const breakLine = document.createElement('hr');
+  const inputTitle = document.querySelector('.title');
+  const inputAuthor = document.querySelector('.author');
+  titleBook.innerHTML = inputTitle.value;
+  authorBook.innerHTML = inputAuthor.value;
```

Comment:
- [ ] [OPTIONAL] It will give a good user experience if the form is reset(input fields cleared) when a book is added.